test_vm/vendor/cookbooks/dbfit_test/recipes/selinux.rb

```diff
@@ -0,0 +1,9 @@
+selinux_state 'SELinux Permissive Temp' do
+  temporary true
+  action :permissive
+end
+
+selinux_state 'SELinux Permissive Perm' do
```

Comment:
I was also thinking that I should leave out the `temporary false` for the permanent mode change as `false` is the default but leaving in makes it more obvious. What's your view?